### PR TITLE
Add qoute-props as needed rule.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Comment by aljones15** [comment]
it's probably not a good idea to do this until someone has taken this branch for a spin.
remember that func-parens consistent thing? we should use this in a few branches first and make sure it's not super annoying.

File: index.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = {
     "operator-linebreak": ["error", "after"],
     "prefer-const": "error",
     "quotes": ["error", "single"],
+    "qoute-props": ["error", "as-needed"],
```
Comment (dlongley) [inline index.js]:
```suggestion
    "quote-props": ["error", "as-needed"],
```

Comment (aljones15) [reply index.js]: good catch

**Comment by davidlehn** [comment]
Fixes https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/13

**Comment by aljones15** [comment]
lol so I just added this to veres-wallet and it causes the ruleset to fail itself so I will correct the rule set

**Comment by aljones15** [comment]
Fixes: https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/15
Provided that it is an error =)

**Pull request merged**